### DISCUSSION TITLE
Error while rendering entity in world

### DISCUSSION THREAD

**Discussion by Delfite** [FabricMC/fabric-api#2458] [Player support] [answered]
Latest.log: https://gnomebot.dev/paste/1006738860736266362

MC Version: 1.19.2
Fabric Version: 0.14.9
API Version: 0.59.0

Description: I was mining Iron in a cave on a survival server when my game crashed and threw an error at me. I didn't have any shaders active when the crash happened, but I did have Tweakeroo's Gamma override setting enabled so I could see in the dark (though I doubt that's what caused the crash).

What should've happened: My shouldn't have crashed seemingly out of nowhere.

What actually happened: 😠

**Comment by warjort** [comment, marked as answer]
````
Caused by: java.lang.NullPointerException: Cannot invoke "net.fabricmc.fabric.api.renderer.v1.Renderer.meshBuilder()" because the return value of "net.fabricmc.fabric.api.renderer.v1.RendererAccess.getRenderer()" is null

	at me.pepperbell.continuity.impl.client.ProcessingContextImpl.<init>:L19

	at me.pepperbell.continuity.client.model.CTMBakedModel$CTMQuadTransform.<init>:L70
````

This is an error in the continuity mod.

There is an issue with a crash similar to yours here: https://github.com/PepperCode1/Continuity/issues/147
That links to https://github.com/PepperCode1/Continuity/issues/107

If that doesn't solve your problem, you will need to contact the mod author.

**Reply by warjort** [reply]
There's also a big warning about this on the mod page. :-)
https://www.curseforge.com/minecraft/mc-mods/continuity

**Reply by Delfite** [reply]
Yeah I had realized the problem almost immediately after I posted this, I just didn't know how to delete a thread since Github's UI is NOT easy to understand. Thanks for the help regardless though.
